projects/composition/src/app/pages/tree-table-page/tree-table-page.component.html

```diff
@@ -77,7 +77,7 @@
         (editRowBtnClicked)="onEditRowButtonClicked()"
         (actionBtnClicked)="onActionBtnClicked()"
         (dataReloadBtnClicked)="onReloadBtnClicked()"
-        toolbarTitle="Tree table with rows menus, action and data reload buttons. Each row menu has a 'Remove' button, which is currently {{
+        toolbarTitle="Tree table with rows menus, action and data reload buttons. Each row menu has custom button'Remove' button, which is currently {{
```

Comment:
I think this change needs to be reverted

Reply:
you're right, title reverted